extensions/blocks/calendly/edit.js

```diff
@@ -65,6 +68,25 @@ export default function CalendlyEdit( props ) {
 			</>
 		);
 
+	useEffect( () => {
+		if ( ! url || 'link' === style ) {
```

Comment:
As you can change the URL and get it wrong, do you think it would be good to cover checking the URL for the `link` style too? I realise we probably don't want to show the spinner, but it would be good to let them it's invalid.

<img width="274" alt="image" src="https://user-images.githubusercontent.com/96462/75379958-22e28800-58ce-11ea-830f-4cbf2ae8e82d.png">